chains/v5/chains_dev.json

```diff
@@ -4319,6 +4319,34 @@
                 "precision": 12,
                 "priceId": "bifrost-native-coin",
                 "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Bifrost.svg"
+            },
+            {
+                "assetId": 1,
+                "symbol": "GLMR",
+                "precision": 18,
```

Comment:
price id can be added

Reply:
I've added it bellow, but looks like it is better to have it after precision

Reply:
yeah, sorry didn't noticed